tyk-pro/templates/bootstrap-post-install.yaml

```diff
@@ -76,10 +76,25 @@ spec:
           {{- end }}
           - name: TYK_DASHBOARD_DEPLOY
             value: dashboard-{{ include "tyk-pro.fullname" . }}
+          {{- if or .Values.dash.tls .Values.gateway.tls }}
+          - name: CURL_CA_BUNDLE_TYK
+            value: /etc/ssl/certs/{{.Values.certificates.caBundleFileName}}
+          {{- end}}
         volumeMounts:
+          {{- if or .Values.dash.tls .Values.gateway.tls }}
+          - name: ca-pemstore
+            mountPath: /etc/ssl/certs/{{.Values.certificates.caBundleFileName}}
+            subPath: {{.Values.certificates.caBundleFileName}}
+            readOnly: false
+          {{- end }}
           - name: bootstrap-config-volume
             mountPath: /opt/scripts
       volumes:
+        {{- if or .Values.dash.tls .Values.gateway.tls }}
+        - name: ca-pemstore
+          configMap:
+            name: ca-pemstore
```

Comment:
Should we make this configurable?

Reply:
I have thought about it too but my reasoning was that the client should have as fewer things  as possible to do to enable TLS and for it to work. If you consider we need this configurable, mark it as "needing change" and I will do it asap.